src/test/java/Homework19.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Homework19 extends BaseTest {
+    @Test(dataProvider = "invalidCredentials", dataProviderClass = BaseTest.class)
+    public void deletePlaylist() {
+        provideEmail("denise626@hotmail.com");
+        providePassword();
+        clickSubmitBtn();
+
+        WebElement choosePlaylist = driver.findElement(By.cssSelector("[href = '#!/playlist/17356']"));
+        choosePlaylist.click();
+        WebElement deleteSongPlaylist = driver.findElement(By.cssSelector("[id='extraTabAlbum']"));
+        deleteSongPlaylist.click();
+
+        Assert.assertFalse(deleteSongPlaylist.isDisplayed());
```

Comment:
Next, you need to find the locator of this window and check (Assert....)

![image](https://user-images.githubusercontent.com/104761482/198520916-a356dfa8-cadf-4b93-b208-9502b0edb5dd.png)